### PR TITLE
Linting

### PR DESCRIPTION
**Pull request by eliascodes**
~~**MERGE TwinePlatform/eslint-config-twine#10 FIRST**~~
~~**MERGE #64 FIRST**~~

Related to #51 
Related to #25 

Fixes a bunch of linting errors.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
     "typescript": "^2.9.1"
   },
   "devDependencies": {
-    "eslint-config-twine": "github:TwinePlatform/eslint-config-twine#2.0.2",
+    "eslint-config-twine": "github:TwinePlatform/tslint-config-twine#2.1.0",
```
Comment (astroash) [inline package.json]: eslint-config-twine -> tslint-config-twine

File: package.json [hunk 2]
```diff
@@ -61,7 +61,7 @@
     "typescript": "^2.9.1"
   },
   "devDependencies": {
-    "eslint-config-twine": "github:TwinePlatform/eslint-config-twine#2.0.2",
+    "tslint-config-twine": "github:TwinePlatform/tslint-config-twine#2.1.0",
     "istanbul-merge": "^1.1.1",
```
Comment (astroash) [inline package.json]: Is `istanbul-merge` being used? It looks from travis that we're just running two separate testing suites.

**Comment by astroash** [comment]
@eliascodes is it possible to merge our coverage reports with this setup? It seems pointless using codecov if this isn't possible.

**Comment by eliascodes** [comment]
Codecov merges the reports automatically

**Pull request merged**